src/combobox/combobox.component.ts

```diff
@@ -92,6 +97,8 @@ import { filter } from "rxjs/operators";
 				<ng-content></ng-content>
 			</div>
 		</div>
+
+		<div *ngIf="invalid" class="bx--form-requirement">{{ invalidText }}</div>
```

Comment:
Template also needs a template outlet to be displayed :)

Combo box does it for example.

Reply:
Oh right 🙃 , I updated it to include the outlet and an example in the storybook